dubbo-common/src/main/java/org/apache/dubbo/common/beanutil/JavaBeanDescriptor.java

```diff
@@ -24,21 +24,18 @@
 
 public final class JavaBeanDescriptor implements Serializable, Iterable<Map.Entry<Object, Object>> {
 
+    private static final long serialVersionUID = -8505586483570518029L;
+
     public static final int TYPE_CLASS = 1;
     public static final int TYPE_ENUM = 2;
     public static final int TYPE_COLLECTION = 3;
     public static final int TYPE_MAP = 4;
     public static final int TYPE_ARRAY = 5;
-    /**
-     * @see org.apache.dubbo.common.utils.ReflectUtils#isPrimitive(Class)
-     */
```

Comment:
Keep this comment as it be.

Reply:
according to ur advise, i have changed the pr.